.storybook/react/preview.js

```diff
@@ -8,4 +8,4 @@ import { initializeWorker, mswDecorator } from 'msw-storybook-addon';
 initializeWorker();
 
 
-export const decorators = [intlDecorate(), cssBaseline(), mswDecorator()]
+export const decorators = [ clearSessionStorage(), mswDecorator(), intlDecorate(), cssBaseline()]
```

Comment:
* Add the clear session storage to make sure the login state doesn't get kept between stories